auslib/dockerflow.py

```diff
@@ -1,9 +1,15 @@
+import logging
+
 from os import path
 
 from flask import Response, jsonify
 
 from auslib.global_state import dbo
 
+from raven.contrib.flask import Sentry
+
+sentry = Sentry()
```

Comment:
You shouldn't need to do any of the Sentry set-up yourself - it is already set-up by other parts of Balrog for the entire application.